README.md

```diff
@@ -24,11 +24,11 @@ Teloc is simple, compile-time DI framework for Rust inspired by
 > (a client) that would use it. 
 
 ## Highlights
-- **Compile-time** - teloc uses powerful rust type system for lifetime and existing of dependencies checking in 
+- **Compile-time** - teloc uses the powerful rust type system for lifetime and existing of dependencies checking at 
 compile-time. That means that you cannot build your code if you do not register dependencies for all of registered 
-dependencies or lifetime of dependency does not correspondence to requester. If your code is compile, that mean it run!
+dependencies or lifetime of dependency does not correspond to a requester. If your code compiles, that mean it runs!
```

Comment:
This paragraph is still phrased a little strangely. Something like the following makes more sense to me. What do you think?


> teloc uses the powerful rust type system check for the existence of dependencies that have the proper lifetime at compile-time. This means you cannot compile your code if a required dependency has not been registered or if it's lifetime is shorter to what's requested. If your code compiles, that means it runs!

Reply:
Excellent.